src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +29,16 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        List<String> results = words.stream()
+                .filter(w -> w.length()>12)
+                .distinct()
+                .sorted(Comparator.comparing(String::length))
```

Comment:
긴 문자열을 출력해야 하니, 역순 정렬이 필요할 것으로 보이네요!
```suggestion
                .sorted(Comparator.comparing(String::length).reversed())
```